generators/app/templates/boilerplate/eslintrc.js

```diff
@@ -38,26 +39,26 @@ module.exports = {
         'linebreak-style': [
             'off'
         ],
-        quotes: [
-            'error',
-            'single'
-        ],
+        // quotes: [
+        //     'error',
+        //     'single'
+        // ],
         semi: [
             'error',
             'always'
         ],
         "no-console": [
             'off'
         ],
-        "max-statements": [
-            'error',
-            {
-                'max': 10
-            },
-            {
-                'ignoreTopLevelFunctions': false
-            }
-        ],
+        // "max-statements": [
```

Comment:
This is another one that generates false positives, especially on our test suites. TSLint allows us to specify that it should ignore the mocha grouping functions (suite, describe, etc. blocks), whereas eslint does not. Disabling this rule in eslint eliminates the need to add a disable line to each and every nested suite